transport/http2_client.go

```diff
@@ -98,6 +100,11 @@ type http2Client struct {
 
 	creds []credentials.PerRPCCredentials
 
+	// activity counter
+	activity *uint64
```

Comment:
It's common to just use an `uint64`, rather then a reference. For example see http.Server code: https://github.com/golang/go/blob/7dc97d9e328edc800e2ce41d5b211ef4e0ef41d0/src/net/http/server.go#L2363-L2364.